flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/StreamWindowSQLExample.java

```diff
@@ -60,7 +62,6 @@ public static void main(String[] args) throws Exception {
 		tempFile.deleteOnExit();
```

Comment:
nit: could we move those 4 helper lines into a method like "String path = createTempFile(orders)". They are actually not important for understanding Flink and should not me moved down.